command/repo.go

```diff
@@ -145,6 +138,17 @@ func repoClone(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func addUpstreamRemote(parentRepo ghrepo.Interface, cloneURL string) error {
```

Comment:
this naming is confusing to me since it sounds like it's just talking about adding an entry with `git remote add`. If it's also going to clone maybe something like `createUpstreamRemote` would be better.

Reply:
Hmm, this function only literally adds a remote with `git remote add`, so I feel its name is in order. Cloning is done outside of the function, but `cloneURL` is passed so that the function knows how to `cd` into the proper git directory before executing `git remote add` (this is done via the `-C <dir>` git flag)

Reply:
i...completely misread this code

nevermind and carry on